EECS3311-PD1/src/test/java/FetchersTest.java

```diff
@@ -0,0 +1,14 @@
+import Fetchers.AbstractFetcher;
+import Fetchers.HospitalBedsFetcher;
+import org.junit.jupiter.api.Test;
+
+public class FetchersTest {
+
+    @Test
+    public void testing(){
+        AbstractFetcher fetcher = new HospitalBedsFetcher("2000", "2001", "CA");
+    }
+
+    
+
+}
```

Comment:
Already implemented in [EECS3311-PD1/src/test/java/Fetchers/FetchersTest.java](https://github.com/4rgc/EECS3311-PD1/blob/9a22bf6164b49b9d714463a73e7cdf36fa07ede7/EECS3311-PD1/src/test/java/Fetchers/FetchersTest.java)